schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,163 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "description": "The configuration file for the Snowplow mobile trackers.",
+  "type": "object",
+  "properties": {
+    "configurationVersion": {
+      "description": "Version number that identifies the current configuration. It has to be increased on each update.",
+      "type": "integer",
+      "minimum": 0
+    },
+    "configurationBundle": {
+      "description": "The list of configurations for each tracker to configure.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "namespace": {
+            "description": "The namespace string that identifies the single tracker.",
+            "type": "string"
+          },
+          "networkConfiguration": {
+            "description": "Represents the network communication configuration allowing the tracker to be able to send events to the Snowplow collector.",
+            "type": "object",
+            "properties": {
+              "endpoint": {
+                "description": "URL of the collector that is going to receive the events tracked by the tracker. The URL can include the schema/protocol (e.g.: http://collector-url.com). In case the URL doesn’t include the schema/protocol, the HTTPS protocol is automatically selected.",
+                "type": "string"
+              },
+              "method": {
+                "description": "The method used to send the requests (GET or POST).",
```

Comment:
Since this is not required, perhaps the default should be added in the description?

Reply:
I think it depends where we enforce the default value. At the moment the default value is set by the tracker. A different tracker version could change the default value without invalidate the correctness of the schema.
Same point would value for all the settings of the other configurations (Especially TrackerConfiguration). Unless we think that a new major version of the tracker (with breaking changes in the default values) would force an update of this schema too.